contributing.md

```diff
@@ -36,74 +36,43 @@ If the code which you are working on does not yet have any documentation, we wou
 - Python code should generally follow the *PEP8* style. We use some custom naming conventions (see below) to have our notation follow the Gaussian process literature. Use `pylint` and `mypy` for formatting and _type checking_. GPflow project has a `.pylintrc` with some relaxed naming conventions.
 - Practise writing good code as far as is reasonable. Simpler is usually better. Reading the existing GPflow code should give a good idea of the expected style.
 
-Example:
-
-```python
-class Foo:
-    """This is an example class with a single simple static method.
-    It mimics a singleton class which can run TensorFlow tensors."""
-
-    @classmethod
-    def add_one(cls, tensor: tf.Tensor, op_name: Optional[str] = None) -> Union[int, float, np.ndarray]:
-        """Increment input tensor and run it in default GPflow session.
-        It is assumed that you have already instantiated variables in 
-        the default GPflow session before calling this method.
-
-        :param tf.Tensor tensor: Input tensor.
-        :param str op_name: Name scope for TensorFlow operation.
-
-        :return: Result depends on which value was passed to
-            the method. It can be either a scalar or NumPy array.
-        """
-        default_name = self.__class__.__name__
-        name = op_name if op_name is not None else default_name
-        session = gpflow.get_default_session()
-        with tf.name_scope(name):
-            incr_tensor = tensor + 1
-        return session.run(incr_tensor)
-```
-
```

Comment:
I don't think this example adds enough to warrant the effort of updating it to tensorflow2. Might be better to point to some actual code example inside gpflow that we think showcases our standard particularly well.

Reply:
I agree with removing this example. The code as a whole may speak louder than pointing to a particular example. If we want to champion a certain level of quality, we'd be best off walking our talk.